doc-content/enterprise-only/employee-rostering/optashift-ER-building-eclipse-proc.adoc

```diff
@@ -3,9 +3,10 @@
 You can use Eclipse, including Red Hat JBoss Development Studio, to build the employee rostering starter application and run it.
 
 .Prerequisites
-. Eclipse is installed.
-. The host has access to the Internet (for downloading Maven packages from external repositories).
-. Google Chrome is installed in order to run the application with the suggested configuration. However, you can modify the configuration to use another web browser.
+* You prepared the deployment files as described in <<er-deploy-prepare-proc>>.
+* Eclipse is installed.
+* The host has access to the Internet (for downloading Maven packages from external repositories).
```

Comment:
Just an observation. The IBM SG says not to use parenthesis unless abs necessary. I would remove these and consider this all of your docs.